pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "notion-app";
+  version = info.version;
+  src = fetchurl { inherit (info) url hash; };
+  sourceRoot = ".";
+  nativeBuildInputs = [ unzip ];
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
+  installPhase = ''
+    mkdir -p $out/Applications
+    mv Notion.app $out/Applications
+  '';
+  passthru.updateScript = ./update/update.mjs;
+  passthru.tests.version = testers.testEqualContents {
+    assertion = "version should be as expected";
+    expected = writeText "expected" finalAttrs.version;
+    actual =
+      runCommand "actual"
+        {
+          nativeBuildInputs = [
+            finalAttrs.finalPackage
+            libplist
+            jq
+          ];
+        }
+        ''
+          plistutil --infile ${finalAttrs.finalPackage}/Applications/*/Contents/Info.plist --format json | jq --raw-output --join-output '.CFBundleShortVersionString' > $out
```

Comment:
Can you add a comment explaining what this does, for the uninitiated?

Reply:
I want to add a version check for zip dist.